tests/test_flagsmith.py

```diff
@@ -99,9 +99,10 @@ def test_get_identity_flags_calls_api_when_no_local_environment_no_traits(
     identity_flags = flagsmith.get_identity_flags(identifier=identifier).all_flags()
 
     # Then
-    assert responses.calls[0].request.body.decode() == json.dumps(
-        {"identifier": identifier, "traits": []}
-    )
+    body = responses.calls[0].request.body
+    if isinstance(body, bytes):
+        body = body.decode()
```

Comment:
I would suggest a short comment here why an `isinstance` check is necessary here.

Reply:
Sure!